specification/netapp/resource-manager/Microsoft.NetApp/stable/2020-07-01/netapp.json

```diff
@@ -0,0 +1,4786 @@
+        }
+      }
+    },
+    "backupPolicy": {
+      "description": "Backup policy information",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "required": [
+        "location",
+        "properties"
+      ],
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "$ref": "#/definitions/resourceTags"
+        },
+        "properties": {
+          "description": "Backup policy Properties",
+          "$ref": "#/definitions/backupPolicyProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "backupPoliciesList": {
+      "description": "List of Backup Policies",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A list of backup policies",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/backupPolicy"
+          }
+        }
+      }
+    },
+    "backupPolicyDetails": {
+      "description": "Backup policy properties",
+      "x-ms-azure-resource": true,
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource Id"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource name"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Resource type"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "$ref": "#/definitions/resourceTags"
+        },
+        "properties": {
+          "description": "Backup policy Properties",
+          "$ref": "#/definitions/backupPolicyProperties",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "backupPolicyPatch": {
+      "description": "Backup policy Details for create and update",
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ]
+        },
+        "id": {
```

Comment:
The delta diff with 06-01 shows these are the main changes. Since this is PATCH definition (line 2423) and these are all read-only and Non-patchable, it is strange they are being added here.

Reply:
Hi,
Looks like this was added to address a oav validation issue found by one of our developers. I will reach out to to verify.